src/etf/openpgp/kv180141dmd180153d/RingCollections.java

```diff
@@ -0,0 +1,36 @@
+package etf.openpgp.kv180141dmd180153d;
+
+import java.io.IOException;
+
+import org.bouncycastle.openpgp.PGPException;
+import org.bouncycastle.openpgp.PGPPublicKeyRing;
+import org.bouncycastle.openpgp.PGPPublicKeyRingCollection;
+import org.bouncycastle.openpgp.PGPSecretKeyRing;
+import org.bouncycastle.openpgp.PGPSecretKeyRingCollection;
+
+public class RingCollections {
```

Comment:
Nadam se da odavde možeš dobiti jedan vektor privatnih i jedan vektor javnih ključeva? Pošto njih serijalizujem za perzistenciju. Naravno, nadam se da i od vektora ključeva možemo izgraditi ovo?